lib/pos_consensus.go

```diff
@@ -281,7 +284,18 @@ func (fc *FastHotStuffConsensus) handleBlockProposalEvent(
 		)
 	}
 
-	// TODO: Broadcast the block proposal to the network
+	// Broadcast the block to the validator network
+	validators := fc.networkManager.rnManager.GetValidatorIndex().GetAll()
+	for _, validator := range validators {
+		validator.SendMessage(blockProposal)
+	}
```

Comment:
Do we want to do this async so all the validators receive the message at the same time?